index.js

```diff
@@ -43,6 +44,7 @@ app.use(
     domain: "letsmovehomie.com"
   })
 );
+
```

Comment:
When submitting a PR, make sure all lines that are needed to run our server are not commented, or else our droplet in Digital Ocean will not run our server.